marvel-app/features/search-page/SearchPageViewController.swift

```diff
@@ -0,0 +1,276 @@
+                
+        navigationItem.titleView = searchBar
+                
+        view.addSubview(tableView)
+        tableView.anchor(top: view.safeTopAnchor,
+                         left: view.leftAnchor,
+                         bottom: view.safeBottomAnchor,
+                         right: view.rightAnchor)
+        tableView.dataSource = self
+        tableView.register(ListCard.self, forCellReuseIdentifier: "ComicsCell")
+        
+        view.addSubview(initialScreen)
+        initialScreen.anchor(top: view.safeTopAnchor,
+                             left: view.leftAnchor,
+                             bottom: view.safeBottomAnchor,
+                             right: view.rightAnchor)
+        
+        view.addSubview(notFoundView)
+        notFoundView.anchor(top: view.safeTopAnchor,
+                             left: view.leftAnchor,
+                             bottom: view.safeBottomAnchor,
+                             right: view.rightAnchor)
+        
+    }
+    
+    private lazy var searchBar: UISearchBar = {
+        let bar = UISearchBar()
+        bar.placeholder = SearchPageConstants.searchBarHint
+        bar.delegate = self
+        bar.sizeToFit()
+        bar.searchTextField.clearButtonMode = .never
+        bar.tintColor = .red
+        return bar
+    }()
+
+    
+    //VIEWS
+    
+    private lazy var tableView: UITableView = {
+        let table = UITableView()
+        table.translatesAutoresizingMaskIntoConstraints = false
+        table.separatorStyle = .none
+        table.isHidden = true
+        return table
+    }()
+    
+    private lazy var initialScreen: UIView = {
+        let view = UIView()
+        view.addSubviews(initialText, initialImage)
+        view.isHidden = false
+        
+        initialImage.center(inView: view)
+        
+        initialText.anchor(top: initialImage.bottomAnchor,
+                           left: view.leftAnchor,
+                           right: view.rightAnchor,
+                           paddingLeft: SearchPageConstants.infoTextPadding,
+                           paddingRight: SearchPageConstants.infoTextPadding)
+        return view
+    }()
+    
+    private lazy var initialText: UILabel = {
+        let label = UILabel()
+        label.text = SearchPageConstants.initialText
+        label.font = .systemFont(ofSize: SearchPageConstants.initialTextSize, weight: .bold)
+        label.textColor = .black
+        label.textAlignment = .center
+        return label
+    }()
+    
+    private lazy var initialImage: UIImageView = {
+        let image = UIImage(systemName: "book.fill")?.withTintColor(.gray, renderingMode: .alwaysOriginal)
+        let imageView = UIImageView(image: image)
+        imageView.setHeight(SearchPageConstants.initialImageHeight)
+        imageView.setWidth(SearchPageConstants.initialImageWidth)
+        return imageView
+    }()
+    
+    private lazy var notFoundView: UIView = {
```

Comment:
Please consider moving more complicated views to separate files and creating custom uiview